src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,16 +4,21 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
```

Comment:
You should get a logger for a specific class.
```suggestion
    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
```